cmake/modules/platform/toolcfg/xlc.cmake

```diff
@@ -114,9 +114,9 @@ elseif(OMR_OS_LINUX)
 	)
 elseif(OMR_OS_ZOS)
 	set(OMR_ZOS_COMPILE_ARCHITECTURE "7" CACHE STRING "z/OS compile machine architecture")
```

Comment:
If we go ahead with this change, this should change to "10" to match the V2R3 settings.

Reply:
done